app/frontend/styles/_tag.scss

```diff
@@ -1,41 +1,35 @@
-@mixin tag($color: "blue", $filled: true) {
-  white-space: nowrap;
-
-  @if $filled {
-    background-color: govuk-colour($color);
-    color: govuk-colour("white");
-  } @else {
-    background-color: govuk-colour("white");
-    outline: 3px solid govuk-colour($color);
-    color: govuk-colour($color);
-  }
-}
-
 .app-tag--danger {
-  @include tag($color: "red");
```

Comment:
How come the upstream tag colours fail contrast requirements?

Reply:
This mixin is our own mixin and it puts white text on a coloured background. That's very hard to achieve a good contrast for most colours. It's okay for the standard blue phase banner/task list tag. But not for most other things.